changelogs/fragments/1916-job-status-query-on-job-name.yml

```
@@ -0,0 +1,6 @@
+bugfixes:
+  - module_utils/job.py - When searching for a job name, module performed a '*' (find all), then filtered the results.
```

Since this will be customer-facing is better that we don't refer to the internal utilities but rather, have an entry for each module like
```suggestion
  - zos_job_output - When searching for a job name, module performed a '*' (find all), then filtered the results.
```

and same for zos_job_query and zos_job_submit. Is just a change to avoid doing this rephrasing during release phase.